### ISSUE TITLE
install error CiviCRM 5.36.1 on BACKDROP 1.83 via MAMP 6.3 PHP 7.4.12

### ISSUE THREAD

**Issue by punkyard** [civicrm/civicrm-backdrop#137]
Hi, after first and fresh install of backdrop, i copied CiviCRM folder in modules
no error during set up
but: no page will be displayed (SAFARI and EPIC (chrome-like) macos 10.14.6) except the admin page
"
Page non trouvée
Ajouter une redirection d'URL de cette page vers un autre emplacement
Vous êtes ici : Accueil
La page demandée "/backdrop183/backdrop183/civicrm/a/" n'a pas pu être trouvée.
"

I found in line 234 that the root folder is doubled :

if (!defined('CIVICRM_UF_BASEURL')) {
  define( 'CIVICRM_UF_BASEURL'      , 'https://domain.net/bkd/bkd'); <<< here bkd/bkd, instead of just one bkd, (domain.net/bkd/)

I'm posting this issue on the advice of @laryn, from Mattermost forum

**Comment by jenlampton** [comment]
I'm having the same problem. CiviCRM does not work on PHP 7.4.

**Comment by laryn** [comment]
@punkyard Am I right in understanding that your installation is working now after you "undoubled" the base URL in the `civicrm.settings.php` file?

@jenlampton There are [almost 2,000 installations on PHP 7.4](https://stats.civicrm.org/?tab=technology) so it's not quite that black and white. Is your problem literally the same as the one above (with the doubled base url)? If so, does manually correcting the doubled URL solve it?

**Comment by jenlampton** [comment]
> Is your problem literally the same as the one above (with the doubled base url)? If so, does manually correcting the doubled URL solve it?

I don't know any doubled lines, but switching to PHP 7.3 solved the problem. Would the doubled lines work in 7.3 but not 7.4?

**Comment by laryn** [comment]
@jenlampton No, that sounds different. I did see a Q&A on stack exchange about someone who switched PHP versions (to PHP 7.4) and found the system crashed. In their case they just had to delete the `templates_c` cache folder and the site functioned again. The case mentioned was Wordpress but that folder is used for all CMSs:

 - https://civicrm.stackexchange.com/questions/39590/civicrm-does-not-have-permission-to-write-temp-files-halts-the-system

**Comment by punkyard** [comment]
yes it works now, fine. with PHP Version 7.4.18
